network_wrangler/roadwaynetwork.py

```diff
@@ -163,17 +163,17 @@ class RoadwayNetwork(object):
 
     UNIQUE_LINK_KEY = "model_link_id"
     UNIQUE_NODE_KEY = "model_node_id"
-    UNIQUE_MODEL_LINK_IDENTIFIERS = ["model_link_id"]
+    UNIQUE_MODEL_LINK_IDENTIFIERS = [UNIQUE_LINK_KEY]
```

Comment:
In the example test [`test_kwarg_instantiation()`](https://github.com/wsp-sag/network_wrangler/blob/06740ad41b26418c7ef1276260fc9e59c9b0df0f/tests/test_roadway.py#L1252C8-L1252C8) in test_roadway.py, we update the `UNIQUE_LINK_KEY` to `"different_link_id"`, which works, but it does not automatically update the reference of `UNIQUE_LINK_KEY` in the class, i.e., the `UNIQUE_MODEL_LINK_IDENTIFIEDS` here would still be `["model_link_id"]`, which is causing the test using "links_DiffIDs.json" to fail.

We need to refresh all class attrs whenever setattr() is called.